docs/advanced/data_types.mdx

```diff
@@ -93,6 +93,16 @@ app.add("my-page-cfbc134ca6464fc980d0391613959196", "notion")
 app.add("https://www.notion.so/my-page-cfbc134ca6464fc980d0391613959196", "notion")
 ```
 
+### Wikipedia
+
+To add any wikipedia page, use the data_type as `wikipedia`. Eg:
```

Comment:
Either you find a way to detect this data_type in `utils.py` or you should include a note that it's not detected ([https://github.com/embedchain/embedchain/pull/471/files#diff-db10e716eaa5a242a1d9552d79ce1980bcc8e2a9b961b4cca94cc056a190c524R78](example)).

Auto detecting could be hard, I mean we could use it as a last step and then try/catch to see if it's a wikipedia page and if not fallback to text (our fallback), but that would add network latency to each detection that falls back to text.